sphere-common/src/main/java/io/sphere/sdk/search/EntitySearch.java

```diff
@@ -18,7 +18,7 @@
 
     List<FilterExpression<T>> facetFilters();
 
-    List<SearchSort<T>> sort();
+    List<SortExpression<T>> sort();
```

Comment:
Yes, this one affects outside the experimental Model, but deprecating can be too painful. On the other side, using the model should be making the change transparent for the dev. I will document it well in the RN.